src/components/BShuttle/BShuttle.vue

```diff
@@ -0,0 +1,480 @@
+          @keyup.space.prevent.stop="onOptionClick(option, 'available', 'chosen', $event)"
+          @keyup.down.prevent.stop="onKeyupDown($event, option, 'available', 'chosen')"
+          @keyup.up.prevent.stop="onKeyupUp($event, option, 'available', 'chosen')"
+          @keyup.right.prevent.stop="onMoveSelectedClick('available')"
+          @keyup.enter.prevent.stop="onMoveSelectedClick('available')"
+          @keydown.up.prevent.stop
+          @keydown.down.prevent.stop
+          @keydown.space.prevent.stop
+          role='option'
+          tabindex='0'
+        >
+        {{ option.displayText }}
+        </li>
+      </ul>
+    </div>
+
+    <div class='buttons'>
+      <button 
+        @click="onMoveAllOptions('available')"
+        id='all-to-chosen'
+        type='button'
+        >
+        <img class="icon" :src="allToChosen" />
+      </button>
+      <button 
+        @click="onMoveSelectedClick('available')"
+        id='selected-to-chosen'
+        type='button'
+        >
+        <i class='icon ion-arrow-right-c' aria-hidden='true'></i>
+      </button>
+      <button
+        @click="onMoveSelectedClick('chosen')"
+        id='selected-to-available'
+        type='button'>
+        <i class='icon ion-arrow-left-c' aria-hidden='true'></i>
+      </button>
+      <button 
+        @click="onMoveAllOptions('chosen')"
+        id='all-to-available'
+        type='button'
+        >
+        <img class="icon" :src="allToAvailable" />
+      </button>
+    </div>
+
+    <div class='chosen'>
+      <label id='chosen-label'>{{ chosenLabel }}</label>
+      <ul
+        aria-labeledby='chosen-label'
+        role='selectbox'
+      >
+        <li
+          v-for="option in chosenOptions"
+          :key="'chosen-option-' + id + '-' + option.id"
+          :id="'chosen-option-' + id + '-' + option.id"
+          :class="optionsClass(option, 'chosen')"
+          @click="onOptionClick(option, 'chosen', 'available', $event)"
+          @keyup.space.stop.prevent.stop="onOptionClick(option, 'chosen', 'available', $event)"
+          @keyup.down.prevent.stop="onKeyupDown($event, option, 'chosen', 'available')"
+          @keyup.up.prevent.stop="onKeyupUp($event, option, 'chosen', 'available')"
+          @keyup.left.prevent.stop="onMoveSelectedClick('chosen')"
+          @keyup.enter.prevent.stop="onMoveSelectedClick('chosen')"
+          @keydown.up.prevent.stop
+          @keydown.down.prevent.stop
+          @keydown.space.prevent.stop
+          role='option'
+          tabindex='0'
+        >
+          {{ option.displayText }} 
+
+        </li>
+      <span v-show="errors.has(validationId)" class="error-text" :id="'error-' + id">{{ errors.first(validationId) }}</span>
+      </ul>
+
+    </div>
+
+  </div>
+  
+</template>
+
+<script>
+
+import validationMixIn from '../../mixins/validation'
+
+export default {
+  name: 'b-shuttle',
+  mixins: [validationMixIn()],
```

Comment:
You have to call the function? That doesn't seem right according to their docs.

Reply:
our mixin is a function, not an object